packages/react-router/tests/link.test-d.tsx

```diff
@@ -242,11 +236,9 @@ test('when navigating from a route with no params to a route with params', () =>
 
   const params = expectTypeOf(TestLink).parameter(0).toHaveProperty('params')
 
-  params
-    .exclude<Function | boolean>()
-    .branded.toEqualTypeOf<{ invoiceId: string }>()
+  params.exclude<Function | boolean>().toEqualTypeOf<{ invoiceId: string }>()
 
-  params.returns.branded.toEqualTypeOf<{ invoiceId: string }>()
```

Comment:
We should always `Expand` instead of relying on `branded` in tests. It means we have complicated intersections